corgidrp/combine.py

```diff
@@ -0,0 +1,91 @@
+"""
+Module to support frame combination
+"""
+import numpy as np
+import corgidrp.data as data
+
+
+def combine_images(data_subset, err_subset, dq_subset, collapse="lower"):
+    """
+    Combines several images together
```

Comment:
It looks to me like the operations below try to maintain the total number of counts, which isn't the intuitive mean or median behavior that you'd expect. I think we might want the option to multiply by tot_frames or not and to describe that more in the docstring.

Reply:
I added a second keyword to toggle the scaling by tot_frames